src/components/editor/PropertyRemark.jsx

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+export class PropertyRemark extends Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  render () {
+    try {
+      const url = new URL(this.props.remark)
```

Comment:
based on Michelle's latest comment in #315, this should first check to see if it's a URL.  If so, do what is currently here;  if not, make it a tool tip.   

Also:  when the property remark is absent or an empty string ... can we just not have anything?  we'll already have a property label, no?